src/main/java/com/ldtteam/structurize/management/Structures.java

```diff
@@ -170,95 +158,6 @@ private static void loadStyleMaps()
         }
     }
 
-    /**
-     * load the schematics from the jar.
-     */
-    private static void loadStyleMapsJar()
-    {
-        URI uri;
-
-        for (final String origin : StructureLoadingUtils.originFolders)
-        {
-            try
-            {
-                final URL url = Structures.class.getResource(SCHEMATICS_ASSET_PATH + origin);
-                uri = url.toURI();
-            }
-            catch (@NotNull final URISyntaxException e)
-            {
-                Log.getLogger().error("loadStyleMaps : ", e);
-                return;
-            }
-
-            if ("jar".equals(uri.getScheme()))
```

Comment:
Don't we need this anymore? I know that a lot of this was compatibility for different OS

Reply:
+ Different origin for dev env and for real game